src/infohud.asm

```diff
@@ -1896,6 +1924,9 @@ NumberGFXTable:
     dw #$0C09, #$0C00, #$0C01, #$0C02, #$0C03, #$0C04, #$0C05, #$0C06, #$0C07, #$0C08
     dw #$0C45, #$0C3C, #$0C3D, #$0C3E, #$0C3F, #$0C40, #$0C41, #$0C42, #$0C43, #$0C44
 
+HexGFXTable:
```

Comment:
This is a bad spot for this table, it will affect number overflow.  Try fighting MB3 with MB HP option and you'll see why.

Either move this to the top or to the bottom of existing tables.

Reply:
Or just make everything awesome, that works too :)

Reply:
I thought the new submenus were going to be the star of this show, but the number are really making me happy. Thanks for pointing this out.